.github/workflows/deploy.yml

```diff
@@ -0,0 +1,26 @@
+name: Build eslint-config-scratch
+
```

Comment:
After merging my GHA changes in VM, I discovered that I needed more permissions to let `semantic-release` do its whole job:
https://github.com/scratchfoundation/scratch-vm/blob/cfbc024442e75aea995d1deadb1b6d03cefda7eb/.github/workflows/ci-cd.yml#L12-L16